commands/host.vegman

```diff
@@ -224,6 +224,90 @@ function cmd_virtualmedia_umount {
   usb-ctrl eject "usb.ms.cli"
 }
 
+
+function config_default_set {
+  local yes=""
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      -y | --yes) yes="y";;
+      *) abort_badarg "$1";;
+    esac
+    shift
+  done
+
+  echo "Host BIOS settings will be reset to default on next boot."
+  if [[ -z "${yes}" ]]; then
+    confirm
+  fi
+
+  ipmitool chassis bootdev none clear-cmos=yes >/dev/null
+}
+
+function config_default_cancel {
+  local yes=""
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      -y | --yes) yes="y";;
+      *) abort_badarg "$1";;
+    esac
+    shift
+  done
+
+  echo "Cancel reset to default host settings on next boot."
+  if [[ -z "${yes}" ]]; then
+    confirm
+  fi
+
+  busctl set-property xyz.openbmc_project.Settings \
+         /xyz/openbmc_project/control/host0/boot/one_time \
+         xyz.openbmc_project.Control.Boot.ClearCmos ClearCmos b false
+}
+
+function config_default_status {
+  expect_noarg "$@"
+  if (ipmitool chassis bootparam get 5 | grep -i 'CMOS Clear'>/dev/null); then
+    echo "BIOS settings will be RESET to defaults on next boot"
+  else
+    echo "BIOS settings will NOT be reset to defaults"
+  fi
+}
+
+# @sudo cmd_config admin,operator,user
+# @doc cmd_config
+# Host configuration
+function cmd_config {
+  subcommand "$@"
+}
+
+# @sudo cmd_config_default admin
+# @restrict cmd_config_default admin
+# @doc cmd_config_default
+# Host set/reset UEFI boot configuration on next boot.
+#  --cancel  - Undo reset UEFI boot configuration, if it was set by mistake
+#  --status  - Shows will be or not UEFI reset during next boot
+#  -y, --yes - Do not ask for interactive confirmation
+function cmd_config_default {
+  local yes=""
+  local action="set"
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      --cancel) action="cancel";;
+      --status) action="status";;
+      -y | --yes) yes="-y";;
+      *) abort_badarg "$1";;
+    esac
+    shift
+  done
+
+  case "$action" in
+    cancel) config_default_cancel $yes;;
+    status) config_default_status;;
+    set)    config_default_set $yes;;
+    *) echo "Unsupported parameter. Use --status, --cancel or --yes to
+approve";;
```

Comment:
Please don't indent like this. Close the quote and use line continuation `\` to do proper indentation:

```
echo "Long string starts here" \
     "and continues on the next line" \
     "with proper indentation."
```